### ISSUE TITLE
Add Issue Labeler [Github Action]

### ISSUE THREAD

**Issue by atarax665** [Tejas1510/Hacking-Scripts#85]
## Is your feature request related to a problem?

With so many new issues kicking in everyday it will get harder for the project maintainer/ moderators to label each and every issue once the repository grows.

## Describe the solution you'd like

You can come up with a github action to label all the issues with a general set of labels or either  based on the keywords used in the issue description etc, so that moderators would have to only add utmost 2-3 specific labels everytime. You can come up with your own creativity to achieve this task, you may also check out some existing issue labelers and integrate one of them to this repository.

You can mention me (@atarax665) once you raise your PR.

**Comment by Tejas1510** [comment]
That seems to be a great issue
You can work on this @atarax665

**Comment by tripathyprateek** [comment]
yeah. Just add a [label] tag inside .github/issue_template and that would be generated automatically when a issue is created. Or another thing we can do is add issue template with various languages, and the person who create the issue can add tags later.

**Comment by atarax665** [comment]
> That seems to be a great issue
> You can work on this @atarax665

I have created this issue for other contributors. I hope someone with take this up soon.

**Comment by atarax665** [comment]
> yeah. Just add a [label] tag inside .github/issue_template and that would be generated automatically when a issue is created. Or another thing we can do is add issue template with various languages, and the person who create the issue can add tags later.

@tripathyprateek Yeah, and there are many issue labelers available that use Regex to label issues. I personally find them more effective then others.
